src/pages/Main/component/RecentSearch/RecentSearch.js

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+
+const RecentSearch = () => {
+  const navigate = useNavigate();
+  const recentData = JSON.parse(localStorage.getItem('recentData'));
+  const recentThree = [
+    recentData[recentData.length - 3] && recentData[recentData.length - 3],
+    recentData[recentData.length - 2] && recentData[recentData.length - 2],
+    recentData[recentData.length - 1] && recentData[recentData.length - 1],
```

Comment:
```suggestion
    recentData[recentData.length - 3],
    recentData[recentData.length - 2],
    recentData[recentData.length - 1],
```
- && 연산자는 없어도 되겠습니다.